File: Java/eoo.java
import java.util.Scanner;
public class eoo{
    public static void main(String[]args){
        while(true){
        System.out.println("Enter a number");
        Scanner s=new Scanner(System.in);
        int a=s.nextInt();
        if(a%2==0)
        {
            System.out.println(a+" is an even number");
        }
        else 
        {
            System.out.println(a+" is an odd number");
        }
    }
}
}

    

File: C/eoo.c
#include <stdio.h>
int main()
{
    printf("Enter the number to check if it is even or odd\n");
    int a;
    scanf("%d", &a);
    if (a % 2 == 0)
        printf("%d is an even number\n", a);
    else
        printf("%d is an odd number", a);
    return 0;
}